cli/build.gradle.kts

```diff
@@ -17,33 +17,20 @@
  * License-Filename: LICENSE
  */
 
-import com.google.cloud.tools.jib.gradle.JibTask
-
 import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 
-val dockerImagePrefix: String by project
-val dockerImageTag: String by project
-
 plugins {
     id("ort-server-kotlin-multiplatform-conventions")
     id("ort-server-publication-conventions")
 
     // Apply third-party plugins.
-    alias(libs.plugins.jib)
     alias(libs.plugins.kotlinSerialization)
 }
 
 group = "org.eclipse.apoapsis.ortserver.cli"
 
-tasks.withType<JibTask> {
-    notCompatibleWithConfigurationCache("https://github.com/GoogleContainerTools/jib/issues/3132")
-}
-
 kotlin {
-    jvm {
-        // Include Java sources into the JVM target's compilations, to make them available for the Jib plugin.
-        withJava()
-    }
+    jvm()
```

Comment:
This can be removed. It is part of the `ort-server-kotlin-multiplatform-conventions` plugin.